discv5/discv5-wire.md

```diff
@@ -291,6 +295,26 @@ TOPICQUERY requests nodes in the [topic queue] of the given topic. The recipient
 request must send one or more NODES messages containing node records registered for the
 topic.
 
+## Protocol Notifications
+
+This section lists all defined notifications which can be sent and received. The hexadecimal
+value in parentheses is the `notification-type`.
+
+### RELAYINIT (0x01)
```

Comment:
It seems you define 'messages' and 'notifications' as two separate things. I think it would be better to have a single sequence for message numbers.